scripts/helm3-init

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DEPLOY_ROOT_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+export DEPLOY_ROOT_DIR
+source "$DEPLOY_ROOT_DIR/src/common.bash"
+
+helm3-init() {
+  if [ -z "${KUBE_NAMESPACE}" ]; then
+    echo "KUBE_NAMESPACE is missing."
+    exit 1
+  fi
+  kubectl create ns $KUBE_NAMESPACE || true
+  helm3 repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
is this needed ?

Reply:
Mi hai chiesto e poi l'hai mergiata comunque 😅 

Lo facciamo con helm 2, quindi suppongo che qui ci sia qualche chart di qualche dipendenza nei nostri chart, però in effetti non ho indagato se lo fosse.